### ISSUE TITLE
Not able to obtain the same sensitivity values when trained on COVIDNet-CXR-Large model

### ISSUE THREAD

**Issue by ShraddhaGS** [lindawangg/COVID-Net#56]
I am not getting the same sensitivity values when trained for 15 more epochs. The sensitivity values are not retained for the 1st epoch itself!
I have used COVIDNet-CXR-Large model and the  dataset files being train_COVIDx2.txt and test_COVIDx2.txt. 
It has been mentioned in the paper that you have used a learning rate policy which reduces the learning rate if the learning is stagnated for a period of time. The factor and patience values, 0.7 & 5 respectively have also been mentioned in the paper. However, I did not come across any line in the code which implement this. 
I have tried to train the model on the same dataset for another 30 epochs as well with different learning rates (2e-07 & 2e-08). The sensitivity kept dropping. 
Am I missing something?

**Comment by tgdwcj** [comment]
hello, I want to ask the way to run COVID-CXR-Large,do you use pb or ckpt?
